desktop/AccountState.ts

```diff
@@ -1,39 +1,43 @@
 import path from 'path';
 import fs from 'fs';
 import { app } from 'electron';
-import { AccountBalance } from '../shared/types';
-import { Tx, Reward } from '../shared/types/tx';
+import { AccountBalance, Tx, Reward } from '../shared/types';
 import { debounce } from '../shared/utils';
 import Logger from './logger';
 
 const logger = Logger({ className: 'AccountState' });
 
+interface GenesisIDState<T> {
+  [genesisID: string]: T;
+}
+
+interface GenesisIDStateType {
+  state: Required<AccountBalance>;
+  txs: { [txId: Tx['id']]: Tx };
+  rewards: { [layer: number]: Reward };
+}
+
 // Types
-export interface AccountState {
+export type AccountState = GenesisIDState<GenesisIDStateType> & {
```

Comment:
```suggestion
export type AccountState = Record<HexString, GenesisIDStateType> & {
```
or even add a global-scoped alias `type GenesisID = HexString`?